src/components/secondPassphrase/secondPassphrase.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
-// import Fees from '../../constants/fees';
+
+import { generatePassphrase } from '../../utils/passphrase';
+import FirstStep from './firstStep';
 import MultiStep from '../multiStep';
-// import Info from '../passphrase/info';
-import CreateSecond from '../passphrase/createSecond';
-import Safekeeping from '../passphrase/safekeeping';
-import Confirm from '../passphrase/confirm';
-import Box from '../box';
-import styles from './secondPassphrase.css';
+import SummaryStep from './summaryStep';
+import TransactionResult from '../transactionResult';
 import routes from '../../constants/routes';
+import styles from './secondPassphrase.css';
 
 class SecondPassphrase extends React.Component {
+  constructor() {
+    super();
+
+    // istanbul ignore next
+    const crypotObj = window.crypto || window.msCrypto;
+    this.secondPassphrase = generatePassphrase({
```

Comment:
I think this function will looks better in a utils file and then just called in here

Reply:
Good catch. Fixed.